services-core/src/gas_price/priority.rs

```diff
@@ -17,10 +39,21 @@ impl PriorityGasPrice {
         T: Fn(&'a dyn GasPriceEstimating) -> F,
         F: Future<Output = Result<f64>>,
     {
-        for estimator in &self.estimators {
-            match operation(estimator.as_ref()).await {
-                Ok(result) => return Ok(result),
-                Err(err) => log::error!("gas estimator failed: {:?}", err),
+        const ORDERING: Ordering = Ordering::SeqCst;
```

Comment:
ubernit: I think you ended up having to type more to use the constant :smile:. Also, I would slightly prefer to inline the ordering with the atomic operations in general, as when reasoning about atomic code, its nice to see the ordering with the atomic operation to more easily reason about how memory is synchronized.